programs/clearing_house/src/controller/orders.rs

```diff
@@ -1048,13 +1048,15 @@ fn fulfill_order(
     let mut quote_asset_amount = 0_u128;
     let mut order_records: Vec<OrderActionRecord> = vec![];
     for fulfillment_method in fulfillment_methods.iter() {
-        if user.orders[user_order_index].status != OrderStatus::Open {
+        let user_order = user.orders[user_order_index];
```

Comment:
i think it's cheaper the way it was before as this requires copying the order into a new variable 🤔